common/src/test/resources/testPayment.properties

```diff
@@ -0,0 +1,10 @@
+# needed for convinience to make parsing easier and more independed of parameters from outside
+methods.interface=test
+
+# comma separated list of available payment method IDs
+methods.available=test-methodid
+
+# config for each method
+methods.test-methodid.method=test-methodid
```

Comment:
I would put whole configuration into a JSON instead of properties and load from filesystem from JSON to model. You can then reuse this model/JSON if you save it as custom-object.